com.openwall.John.desktop

```diff
@@ -1,5 +1,5 @@
 [Desktop Entry]
-Version=1.9.0J1
+Version=v1-rolling
```

Comment:
this is not what `Version` is about in the .desktop file. It was already wrong before.